src/io/eval.rs

```diff
@@ -56,48 +55,22 @@ where
     let bgg_pubkey_sampler = BGGPublicKeySampler::<_, SH>::new(hash_key, d);
     let public_data = PublicSampledData::<SH>::sample(&obf_params, hash_key);
     log_mem("Sampled public data");
-
     let packed_input_size = public_data.packed_input_size;
+    let m_b = (1 + packed_input_size) * (d + 1) * (2 + log_base_q);
```

Comment:
Why did you change the defintion of `mb` ?

Reply:
from my math its correct

Reply:
I think this is correct.
Ref. Algorithm 11 of https://eprint.iacr.org/2018/1222.pdf